src/lib/crypto/kimchi_backend/common/tests/test_lookup_table_constraint_kind.ml

```diff
@@ -37,15 +39,391 @@ let test_finalize_and_get_gates_with_lookup_tables () =
 let test_finalize_and_get_gates_with_runtime_table_cfg () =
   let module Tick = Kimchi_backend.Pasta.Vesta_based_plonk in
   let cs = Tick.R1CS_constraint_system.create () in
+
+  let indexed_runtime_table_cfg =
+    [| Tick.Field.zero
```

Comment:
`Array.init 4 ~f:Tick.Field.of_int` would be cleaner

Reply:
Fixed in d20bbe026caf80248d79979fcfbbd79f26ff2b23